dart/lib/src/sentry_client.dart

```diff
@@ -278,6 +285,10 @@ class SentryClient {
 
   void close() => _options.httpClient.close();
 
+  void recordLostEvent(DiscardReason reason, DataCategory category) {
```

Comment:
I believe this should not be a public method, cc @philipphofmann

Reply:
No, this should be internal.